google-cloud-tasks/samples/create_http_task.rb

```diff
@@ -0,0 +1,85 @@
+# Copyright 2018 Google, Inc
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START cloud_tasks_create_http_task]
+require "google/cloud/tasks"
+
+# Create a Task with an HTTP Target
+#
+# @param [String] project_id Your Google Cloud Project ID.
+# @param [String] location_id Your Google Cloud Project Location ID.
+# @param [String] queue_id Your Google Cloud Tasks Queue ID.
+# @param [String] url The full path to sent the task request to.
+# @param [String] payload The request body of your task.
+# @param [Integer] seconds The delay, in seconds, to process your task.
+def create_http_task project_id, location_id, queue_id, url, payload: nil, seconds: nil
+  # Instantiates a client.
+  client = Google::Cloud::Tasks.new
```

Comment:
The code here needs to be updated to conform to the changed usage currently on git master. See `google-cloud-tasks/MIGRATING.md` for details. But the TL;DR is:
* This is now `Tasks.cloud_tasks` instead of `Tasks.new`
* The `queue_path` call below takes keyword arguments instead of positional arguments. See `google-cloud-tasks-v2/lib/google/cloud/tasks/v2/cloud_tasks/paths.rb` for the definition of the method.
* The `create_task` call also takes keyword arguments. See `google-cloud-tasks-v2/lib/google/cloud/tasks/v2/cloud_tasks/client.rb` for the definition of the method.

To test, you'll need to run tests against master (rather than the released library) by setting `GOOGLE_CLOUD_SAMPLES_TEST=master` (see your Gemfile).